constance/utils.py

```diff
@@ -1,6 +1,27 @@
+from __future__ import print_function
+from __future__ import unicode_literals
+
 from importlib import import_module
 
+from constance import LazyConfig, settings
+
+config = LazyConfig()
```

Comment:
I think instantiating another config instance isn't needed here anymore since the reason it was done originally was to prevent a circular import in the admin module. You could in theory use the `constance.config` lazy instance instead